lib/order.rb

```diff
@@ -0,0 +1,30 @@
+require_relative "menu"
+
+class Order
+
+  attr_reader :basket, :item, :quantity, :basket_total
+
+  def initialize
+    @menu = Menu.new
+    @basket = []
+    @item = item
+    @quantity = quantity
+    @basket_total = 0
+  end
+
+  def add_item(item, quantity)
+    @item = item
+    @quantity = quantity
+  end
+
+  def to_basket
+    item_cost = @menu[@item]
```

Comment:
at the moment this class isnt too dependent on MENU class it may work better to pass menu into class as argument